src/Microsoft.AspNetCore.Hosting.Abstractions/IHostingEnvironment.cs

```diff
@@ -8,20 +8,8 @@ namespace Microsoft.AspNetCore.Hosting
     /// <summary>
     /// Provides information about the web hosting environment an application is running in.
     /// </summary>
-    public interface IHostingEnvironment
+    public interface IHostingEnvironment : Extensions.Hosting.IHostingEnvironment
     {
-        /// <summary>
-        /// Gets or sets the name of the environment. The host automatically sets this property to the value
```

Comment:
These comments might provide valuable, web-specific information which gets lost this way.